feedthefox/devices/views.py

```diff
@@ -1,8 +1,30 @@
 from django.shortcuts import render
 
+import django_filters
+
+from feedthefox.devices.models import Device
+
+
+class DeviceFilter(django_filters.FilterSet):
+    type = django_filters.AllValuesFilter()
+    manufacturer = django_filters.AllValuesFilter()
+
+    class Meta:
+        model = Device
+        fields = ['manufacturer', 'type']
+
+    def __init__(self, *args, **kwargs):
+        super(DeviceFilter, self).__init__(*args, **kwargs)
+
+        # Add "Any" entry to choice fields.
+        for name, f in self.filters.items():
```

Comment:
Choices are dynamic and this field does exactly what we are trying to do with a ``ChoiceFilter``. ``AllValuesFilter`` is a ``ChoiceFilter`` populated dynamically with all the value found per field. On top of that, even with the ``ChoiceFilter`` we will have to manually add ``Any`` choice.